src/main/java/net/glowstone/command/minecraft/SeedCommand.java

```diff
@@ -0,0 +1,30 @@
+package net.glowstone.command.minecraft;
+
+import net.glowstone.command.CommandUtils;
+import org.bukkit.ChatColor;
+import org.bukkit.World;
+import org.bukkit.command.CommandSender;
+import org.bukkit.command.defaults.VanillaCommand;
+
+import java.util.Collections;
+
+public class SeedCommand extends VanillaCommand {
+    public SeedCommand() {
+        super("seed", "Displays the world seed.", "/seed", Collections.emptyList());
+        setPermission("minecraft.command.seed");
+    }
+
+    @Override
+    public boolean execute(CommandSender sender, String label, String[] args) {
+        final World world = CommandUtils.getWorld(sender);
+
+        if (world == null) {
+            sender.sendMessage(ChatColor.RED + "Can not get the world");
```

Comment:
No need to send this error, just return false. It would never happen to a block/entity/console.